packages/core/scripts/deploy.ts

```diff
@@ -0,0 +1,32 @@
+import { Logger } from '@eth-optimism/common-ts'
+import hre from 'hardhat'
```

Comment:
I thought about moving all of the deployment logic into the contracts package b/c that makes more sense to me but didn't seem like a big enough deal to do in this PR. LMK if you think we should do this and I can make a ticket for it.

Reply:
No preference; doesn't seem high priority to me